chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/QuestionableActivityEmailJob.java

```diff
@@ -122,17 +133,33 @@ public void execute(final JobExecutionContext jobContext) throws JobExecutionExc
 
             EmailBuilder emailBuilder = new EmailBuilder(env);
             emailBuilder.recipients(recipients)
-            .subject(subject)
-            .htmlMessage(htmlMessage)
-            .fileAttachments(files)
-            .sendEmail();
+                    .subject(subject)
+                    .htmlMessage(htmlMessage)
+                    .fileAttachments(files)
+                    .sendEmail();
 
         } catch (MessagingException e) {
             LOGGER.error(e);
         }
         LOGGER.info("********* Completed the Questionable Activity Email job. *********");
     }
 
+    private Integer getRangeInDays(final JobExecutionContext jobContext) {
+        Integer range = 7;
```

Comment:
This looks like a candidate for a CONSTANT value to use as default if the range is missing